internal/image.go

```diff
@@ -27,3 +27,42 @@ func ParseImage(repo string, tag string) (*Image, error) {
 		Tag:        tag,
 	}, nil
 }
+
+func GetImagePath(obj map[string]interface{}, serviceName string) []string {
```

Comment:
Since kube-builder is now using the `runtime.RawExtension` type, which @jrhouston has written a helper function for to unmarhshal into a `map[string]interface{}`, I think we can stick to this as it will be compatible with the kube-builder stuff.